python/pdo/test/request.py

```diff
@@ -27,13 +28,18 @@
 import pdo.eservice.pdo_helper as enclave_helper
 import pdo.service_client.enclave as eservice_helper
 import pdo.service_client.provisioning as pservice_helper
+from pdo.service_client.servicedatabase import ServiceDB_Manager
 
 import pdo.contract as contract_helper
 import pdo.common.crypto as crypto
 import pdo.common.keys as keys
 import pdo.common.secrets as secrets
 import pdo.common.utility as putils
 
+import requests
+from urllib.parse import urlparse
+import socket
+
 import logging
```

Comment:
these imports should be removed. not used.

Reply:
yes, will do